src/apps/weblib/typeschema-api/styles.yaml

```diff
@@ -452,6 +452,9 @@ definitions:
             interlacing:
                 $ref: Interlacing
                 nullable: true
+            split:
+                $ref: Length
```

Comment:
If the parameter can only be set in percentage, then the type should reflect that.
`{ type: string, mask: /:number:%/ }` instead of `Length`